README.md

```diff
@@ -16,10 +16,12 @@ This section explains the simplest way to set up a local testing environment and
 
 After getting familiar with the basic scenario, you may replace the test canister with your own one, and use this repo as a skeleton for creating a custom testing environment.
 
+0. Clone your dapp repo into the current directory and `cd` into it.
+
 1. Start a local replica instance:
     ```bash
     SNS_TESTING_INSTANCE=$(
-        docker run -p 8080:8080 -d martin2718/sns-testing:latest dfx start --clean
+        docker run -p 8080:8080 -v `pwd`:/dapp -d martin2718/sns-testing:latest dfx start --clean
```

Comment:
```suggestion
        docker run -p 8080:8080 -v "`pwd`":/dapp -d martin2718/sns-testing:latest dfx start --clean
```
Let's quote the current working directory in case somebody has a space in the name.